README.md

```diff
@@ -444,6 +445,51 @@ Medium.
 - [Idea discussion](https://rust-lang.zulipchat.com/#narrow/channel/421156-gsoc/topic/Idea.3A.20Extend.20testing.20of.20.60std.3A.3Aarch.60.20intrinsics)
 - [t-libs/stdarch](https://rust-lang.zulipchat.com/#narrow/channel/208962-t-libs.2Fstdarch)
 
+### Add safety contracts
+
+**Description**
+
+We have taken a goal to
```

Comment:
"There is a Rust Project goal"

I think that would be a bit clearer.

Reply:
Updated as suggested.